templates/layout/base.html

```diff
@@ -60,6 +60,27 @@
             {{!-- Exported in app.js --}}
             function onThemeBundleMain() {
                 window.stencilBootstrap("{{page_type}}", {{jsContext}}).load();
+
+                function browserSupportsFormData() {
+                    return typeof FormData !== 'undefined' 
+                        && !!FormData.prototype.keys;
+                }
+                function loadFormDataPolyfillScript(src) {
+                    var formDataPolyfillScript = document.createElement('script');
+                    formDataPolyfillScript.src = src;
+                    formDataPolyfillScript.classList.add('formDataClass');
+                    formDataPolyfillScript.onerror = function () {
+                        console.error('Failed to load formData polyfill script ' + src);
+                    };
+                    document.body.appendChild(formDataPolyfillScript);
+                }
+
+                if (!browserSupportsFormData()) {
+                    console.log('formData polyfill works');
+                    loadFormDataPolyfillScript('{{cdn 'assets/dist/theme-bundle.polyfill_form_data.js'}}');
+                } else {
+                    console.log('formData polyfill NOT work');
+                }
```

Comment:
@yurytut1993  does it make sense to have this in the separate js bundle, same as polyfills.js? I know it's just  ~20 lines, but with separate bundle it will be cached in browser and not loaded on every request as html response.

Reply:
We can not guarantee that polyfills.js will be ready before main bundle. 99% that it will be, but still. This is the reason I put it here in base. When I remove extra consoles it will be less lines 😃